packages/core/core/src/AssetGraph.js

```diff
@@ -72,7 +170,7 @@ export default class AssetGraph extends Graph<AssetGraphNode> {
   }
 
   getDependencyResolution(dep: IDependency): ?Asset {
-    let depNode = this.getNode(dep.id);
+    let depNode = this.nodes.get(dep.id);
```

Comment:
why this change?

Reply:
Something was broken with it, I think because I was prepending the node type to the id, but decided to just stop doing that. Thought I put it back to the way it was, good catch.